src/libs/Network/SequentialQueue.ts

```diff
@@ -135,9 +139,23 @@ function process(): Promise<void> {
     const requestToProcess = processNextPersistedRequest();
     if (!requestToProcess) {
         Log.info('[SequentialQueue] Unable to process. No next request to handle.');
+        const wasProcessingRelevantRequest = currentRequestCommand !== null;
+        currentRequestCommand = null;
+        if (wasProcessingRelevantRequest) {
+            notifyQueueStateChange();
+        }
         return Promise.resolve();
     }
 
+    // Track the current request command
+    const previousCommand = currentRequestCommand;
+    currentRequestCommand = requestToProcess.command;
```

Comment:
`currentRequestCommand` isn't the same as `ongoingRequest` from PersistedRequest? to avoid have the same thing in 2 places